rhel7/profiles/stig.profile

```diff
@@ -97,13 +97,15 @@ selections:
     - sudo_remove_nopasswd
     - sudo_remove_no_authenticate
     - accounts_logon_fail_delay
+    - accounts_logon_fail_delay.severity=medium
     - gnome_gdm_disable_automatic_login
     - gnome_gdm_disable_guest_login
     - sshd_do_not_permit_user_env
     - disable_host_auth
     - grub2_password
     - require_singleuser_auth
     - grub2_uefi_password
+    - grub2_uefi_password.severity=high
```

Comment:
This should be updated in the rule itself to match `grub2_password` severity which is `high`

Reply:
done